### PR TITLE
Improve token address page

### LINKED ISSUE #1664: improve /token/{address} page
let's ignore market cap, Official website, social profiles for now.

image

### PR DESCRIPTION
**Pull request by blesstosam**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind backend-change
> /kind ui-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1664 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```
WX20201008-121034@2x
WX20201008-120406@2x

### CODE REVIEW

**Comment by guo** [comment]
Please change page title to 
"{Token Name} ({Symbol}) Token Tracker | iotexscan"

File: src/shared/pages/xrc20-action-list-page.tsx [hunk 1]
```diff
@@ -326,16 +335,214 @@ export const XRC20HoldersTable: React.FC<IXRC20ActionTable> = ({
...
+      type="flex"
+      justify="space-between"
+      align="top"
+      gutter={36}
+      style={{ marginBottom: "36px" }}
+    >
+      <Col xs={24} sm={24} md={12}>
+        <Card
+          title={
+            <span style={{ color: "#333", fontWeight: 400 }}>
```
Comment (anshuomomo) [inline src/shared/pages/xrc20-action-list-page.tsx:362]: we can use 'styled' here. And the `"#333" `-> `colors.XXX`

File: src/shared/pages/xrc20-action-list-page.tsx [hunk 2]
```diff
@@ -326,16 +335,214 @@ export const XRC20HoldersTable: React.FC<IXRC20ActionTable> = ({
...
+              {`${t("token.address.overview")} `}
+              <span
+                style={{ color: "#666", fontWeight: 400, fontSize: "14px" }}
+              >
```
Comment (anshuomomo) [inline src/shared/pages/xrc20-action-list-page.tsx:366]: we can use 'styled' here.

File: src/shared/pages/xrc20-action-list-page.tsx [hunk 3]
```diff
@@ -326,16 +335,214 @@ export const XRC20HoldersTable: React.FC<IXRC20ActionTable> = ({
...
+          <p>
+            <OverviewKey>{t("token.address.transfer")} </OverviewKey>
+            <TotalTransfers tokenAddress={tokenAddress} />
+          </p>
+        </Card>
+      </Col>
+      <Col xs={24} sm={24} md={12}>
+        <Card
+          title={
+            <span style={{ color: "#333", fontWeight: 400 }}>
```
Comment (anshuomomo) [inline src/shared/pages/xrc20-action-list-page.tsx:393]: The same.

**Review by anshuomomo — APPROVED** [review]

**Pull request merged**